source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRContentStackView.h

```diff
@@ -37,6 +40,18 @@
                  priority:(unsigned int)priority
                  location:(ACRBleedDirection)location;
 
+- (UIView *)getLastArrangedSubview;
+
+- (CGFloat)getNextGreatWidth:(UIView *)view;
```

Comment:
I'm not sure what this function does just by its name... should we have comments or a more descriptive name?